src/Command/Pull/PullCommandBase.php

```diff
@@ -896,10 +896,14 @@ protected function environmentPhpVersionMatches($environment): bool {
   }
 
   /**
+   * Get current PHP version using IDE-specific variables when possible.
+   *
+   * Disable mutation testing because we cannot mock PHP constants.
```

Comment:
Is it waiting on https://github.com/infection/infection/issues/315 ? If yes could you link to it?

Reply:
I just shot in the dark and that wasn't it. Ignore 😉